tools/openms/macros.xml

```diff
@@ -13,7 +13,10 @@
     <stdio>
       <exit_code range="1:"/>
       <exit_code range=":-1"/>
-      <regex match="Exception:"/>
+      <regex match="java.lang.OutOfMemoryError" level="fatal_oom" description="Java memory Exception"/>
+      <regex match="Could not allocate metaspace" level="fatal_oom" description="Java memory Exception"/>
+      <regex match="Cannot create VM thread" level="fatal_oom" description="Java memory Exception"/>
+      <regex match="Exception:" level="fatal" description="Java Exception"/>
```

Comment:
The last one seems not very java specific to me. All others I'm fine to merge. Can you just make the description of the last one less Java centric :)

Reply:
the exact output of the adapter (which is as for as I know directly from java) is.

```
Error occurred during initialization of VM
Cannot create VM thread. Out of system resources.
```

So maybe we add `Out of system resources.`?

Reply:
I think its all ok, besides the `Exception:` regex. Exception can be written from everywhere ... and does not need to be a Java issue.

Reply:
OK. I removed it. Even though it was in the original I guess it wont be a problem, since the exit codes should kick in before.

Reply:
And added a new one I just observed in PeakPickerHighRes.